adapters/oidc/js/src/main/resources/keycloak.js

```diff
@@ -438,15 +438,13 @@
                 baseUrl = kc.endpoints.authorize();
```

Comment:
If we're going to make these changes we also need to make sure that the TypeScript definitions are updated. 

The JSDoc on the `scope` field on the `KeycloakLoginOptions` interface must no longer be private and include some descriptive text for the field itself.

And we'll have to add the same `scope` field to the `KeycloakConfig` interface which is used for the configuration passed into the constructor.

Reply:
I eventually found some time to get back to this...
@jonkoops can you point me to the TypeScript definitions that need to be changed for this?

Reply:
Sure, you can find them in the same directory right [here](https://github.com/keycloak/keycloak/blob/master/adapters/oidc/js/src/main/resources/keycloak.d.ts).

Reply:
Hmm, I think this change perhaps has been removed? I don't remember the original code so I guess we can resolve this.